src/Databases/DataLake/DatabaseDataLake.cpp

```diff
@@ -160,10 +160,14 @@ std::shared_ptr<DataLake::ICatalog> DatabaseDataLake::getCatalog() const
         }
         case DB::DatabaseDataLakeCatalogType::ICEBERG_HIVE:
         {
+#if USE_AVRO && USE_HIVE
```

Comment:
Avro is already checked at the beginning of this file, so enough to check only `USE_HIVE`